plasmapy/physics/__init__.py

```diff
@@ -42,3 +42,11 @@
                         mobility,
                         Knudsen_number,
                         coupling_parameter)
+
+from .magnetostatics import (MagneticStatics,
```

Comment:
I'm not sure the main `physics` namespace is the best place for this... Autocompletion is going to be completely useless if we keep piling stuff in there. Maybe it'd be better to keep access to these via `physics.magnetostatics`?